src/main/java/core/db/MatchesKurzInfoTable.java

```diff
@@ -413,4 +414,15 @@ public List<MatchKurzInfo> getMatchesKurzInfo(int teamId, Timestamp  from, Times
 				params.toArray()
 		);
 	}
+
+	public MatchKurzInfo loadMatchesKurzInfo(SourceSystem sourceSystem, int matchId) {
+		var matchTypes = MatchType.fromSourceSystem(sourceSystem);
+		var placeholders =matchTypes.stream().map(i->"?").collect(Collectors.joining(","));
+		var params = new ArrayList<>();
+		params.add(matchId);
+		params.addAll(matchTypes.stream().map(MatchType::getId).toList());
+		var infos =  loadMatchesKurzInfo("WHERE MATCHID = ? AND MATCHTYP IN (" + placeholders + ")", params.toArray());
+		if (!infos.isEmpty()) { return infos.get(0); }
```

Comment:
I suggest to use the standard code formatter from IDEA for the function.

Reply:
changed